index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <a><img src="/images/Welcome.png" alt="welcome button at the top left"></a>
+       <nav> <img src="/images/Union.png" alt="a union of three vectors"> </nav>
+    </header>
+    <br>
+    <section class="second">
+      <h1>  Hey there. I’m Marian
+I’m a software developer</h1>
+    <p>
+    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+    </section>
+            <ul class = socials>
+                <li><a href="https://twitter.com/BadruNdawula"><img src="/images/twitter.jpg" alt=""></a></li>
+                <li><a href="https://www.linkedin.com/in/badru-ndawula-32b371236/"><img src="/images/Linkedin icon.jpg" alt=""></a></li>
+                <li><a href="https://github.com/PrinceBadru"><img src="/images/github.jpg" alt=""></a></li>
+                <li><a><img src="/images/medium.jpg" alt=""><a></li>
+                <li><a><img src="/images/angellist.jpg" alt=""></a></li>
+            </ul>
```

Comment:
- [ ] Although you use the icons, it is not rendering on the web please download the individual images and use them.